### PR TITLE
Ps 5.7 4905

### PR DESCRIPTION
**Pull request by george-lorch**

### CODE REVIEW

**Comment by george-lorch** [comment]
https://ps.cd.percona.com/job/percona-server-5.7-param/76/

File: mysql-test/suite/rocksdb/t/prefix_extractor_override.test [hunk 1]
```diff
@@ -30,17 +30,12 @@ SELECT COUNT(*) FROM t1 WHERE id1=1 AND id2=1 AND id3=1;
...
---echo
---let $restart_parameters=restart:--loose-rocksdb_override_cf_options=cf1={prefix_extractor=capped:26};
---source include/restart_mysqld.inc
-
---echo
---echo Changed Prefix Extractor (after restart):
+--echo Changed Prefix Extractor (after update_cf_options set, without restart):
 --echo
 --sorted_result
+# Restart no longer needed
```
Comment (laurynas-biveinis) [inline mysql-test/suite/rocksdb/t/prefix_extractor_override.test:40]: Perhaps the final restart at the end of testcase is no longer needed neither?

File: mysql-test/suite/rocksdb/t/prefix_extractor_override.test [hunk 2]
```diff
@@ -88,6 +83,7 @@ SELECT COUNT(*) FROM t1 WHERE id1=1 AND id2=1 AND id3=1;
 --let $assert_cond="[select variable_value-@u from performance_schema.global_status where variable_name=\'rocksdb_bloom_filter_prefix_checked\']" = 2
 --source include/assert.inc
 
+SET @@global.rocksdb_update_cf_options = '';
```
Comment (laurynas-biveinis) [inline mysql-test/suite/rocksdb/t/prefix_extractor_override.test:88]: If the final restart is going to be removed, this should be replaced with the usual save/restore idiom

File: mysql-test/suite/rocksdb/t/trx_info_rpl.test [hunk 3]
```diff
@@ -0,0 +1,41 @@
+--source include/have_rocksdb.inc
+--source include/have_binlog_format_row.inc
+--source include/master-slave.inc
+
+connection slave;
```
Comment (laurynas-biveinis) [inline mysql-test/suite/rocksdb/t/trx_info_rpl.test]: Set rpl_skip_start_slave=1 before master-slave.inc instead

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: mysql-test/suite/rocksdb/t/prefix_extractor_override.test (hunk 1 above)
Comment (george-lorch) [reply mysql-test/suite/rocksdb/t/prefix_extractor_override.test:40]: Agreed, the removed restart was there to test through a crash that would have happened on server startup.  This final restart serves no purpose that I can see.

**Comment by george-lorch** [comment]
Suggestions incorporated, new jenkins that only re-tests impacted tests here https://ps.cd.percona.com/job/percona-server-5.7-param/77/

**Review by laurynas-biveinis — APPROVED** [review]
Feel free to upstream MTR changes

**Pull request merged**